envs/public/harbor-config.yaml

```diff
@@ -33,6 +33,7 @@ data:
         harbor:
           annotations:
             cert-manager.io/issuer: letsencrypt
+            nginx.ingress.kubernetes.io/limit-rps: "25"
```

Comment:
Document this change in the `docs` of this project. 
- An example with a real-world scenario could also help to understand how the burst default setting (5) works.

Reply:
See eb378bf06c29cf4e91f5e123d12592598d5bd278, where examples and valuable links show how the nginx rate-limiting works.